src/guiapi/include/st7789v.hpp

```diff
@@ -34,10 +36,6 @@ typedef struct _st7789v_config_t {
     uint8_t control;
 } st7789v_config_t;
 
-#ifdef __cplusplus
-extern "C" {
-#endif //__cplusplus
-
 __attribute__((used)) inline uint16_t color_to_565(uint32_t clr) {
```

Comment:
while switching to C++ this attribute(used) may also disappear

Reply:
fixed